### PR TITLE
Adds .docker/config.json secret example

### PR DESCRIPTION
**Pull request by fabianofranz**
Adds an example about how to create a `.docker/config.json` secret.

@deads2k ptal

### CODE REVIEW

File: pkg/cmd/cli/secrets/new.go [hunk 1]
```diff
@@ -39,7 +39,10 @@ using with all valid keys in that directory.
   $ %[1]s my-secret path/to/bar
 
   // Create a new .dockercfg secret named my-secret
-  $ %[1]s my-secret path/to/.dockercfg`
+  $ %[1]s my-secret path/to/.dockercfg
+
+  // Create a new .docker/config.json secret named my-secret
```
Comment (deads2k) [inline pkg/cmd/cli/secrets/new.go]: New standard uses `#`.  You can make a separate issue to sweep them all if you like.

**Comment by deads2k** [comment]
Comment line format has changed.  You can link a new issue for followup on all commands if you like.

lgtm otherwise.

File: pkg/cmd/cli/secrets/new.go (hunk 1 above)
Comment (fabianofranz) [inline pkg/cmd/cli/secrets/new.go]: Of course, updated.

**Comment by fabianofranz** [comment]
[merge]

**Comment by fabianofranz** [comment]
re[merge]

**Comment by fabianofranz** [comment]
rere[merge]

**Pull request merged**